source/includes/api/catalogues.md

```diff
@@ -326,3 +326,162 @@ minimum_age | integer | when present only products suitable for a minimum age of
+    "name": "LINKS OF LONDON 32-09-006 EFFERVESCENCE PENDANT",
+    "base_price": 158.27,
+    "brand_id": 1,
+    "max_value": 38.95,
+    "description": "<p>A cluster of silver, white and grey pearls come together to create this elegant pendant complete with sterling silver chain. This pendant gives the traditional pearl a modern twist.</p>",
+    "model_no": "LK-016",
+    "sku": "LK-016",
+    "available": true,
+    "availability_note": "",
+    "international_requirements": false,
+    "minimum_age": null,
+    "primary_image_id": 337,
+    "variant": "",
+    "voucher": null,
+    "lowest_denomination": null,
+    "face_value": null,
+    "supplier_currency_id": 31,
+    "available_countries": [
+      {
+        "id": 140,
+        "name": "UK",
+        "vat_rate": "20%",
+        "delivery_charge": 7.2
+      },
+      {
+        "id": 165,
+        "name": "Albania",
+        "vat_rate": "20%",
+        "delivery_charge": 34.75
+      },
+    ...
+    ],
+    "rrp": "159.99",
+    "supplier": {
+      "id": 14
+    },
+    "vat_rate": {
+      "name": "20%",
+      "numeric": 20
+    },
+    "status": {
+      "name": "Approved"
+    },
+    "catalogue": {
+      "id": 8,
+      "name": "Demo Catalogue",
+      "exclusive": false
+    },
+    "categories": [
+      {
+        "id": 62,
+        "name": "Necklaces",
+        "parent_id": 59,
+        "ar_association_key_name": 414
+      }
+    ],
+    "variants": [
+      {
+        "id": 529,
+        "variant": "small",
+        "translations": [
+          {
+            "locale": "fr",
+            "name": "petit"
+          },
+          {
+            "locale": "de",
+            "name": "klein"
+          }
+        ],
+        "sku": "LK-017",
+        "available": true,
+        "product_sku": "LK-016",
+        "face_value": null,
+        "product_id": null,
+        "voucher_status": false,
+        "variant_base_price": null,
+        "face_value_gbp": null
+      },
+      {
+        "id": 530,
+        "variant": "large",
+        "sku": "LK-018",
+        "available": true,
+        "product_sku": "LK-016",
+        "face_value": null,
+        "product_id": null,
+        "voucher_status": false,
+        "variant_base_price": null,
+        "face_value_gbp": null
+      }
+    ],
+    "media": [
+      {
+        "id": 337,
+        "url": "http://redstone-beta.s3.amazonaws.com/2014/10/21/13/38/44/960/lk_016.jpg"
+      }
+    ],
+    "delivery_type": {
+      "id": 1,
+      "name": "Physical",
+      "requires_address": true
+    },
+    "currency": {
+      "title": "Pounds Sterling",
+      "abbreviation": "GBP",
+      "sign": "£"
+    },
+    "translations": [
+      {
+        "locale": "fr",
+        "name": "product_name_french",
+        "description": "product description_french"
+      },
+      {
+        "locale": "de",
+        "name": "product_name_german",
+        "description": "product description_german"
+      }
+    ]
+  },
+  ...
+]
```

Comment:
I think I'd prefer if we didn't use a real products information (just the name, description, skus, and model numbers) or reference http://redstone-beta.s3.amazonaws.com since these are public docs. I can see this is just what was already in for V2 and V3 but it would be good if we could change it